descriptors/ModuleDescriptor-template.json

```diff
@@ -698,6 +698,12 @@
       ]
     }
   ],
+  "requires": [
+    {
+      "id": "configuration",
+      "version": "2.0"
+    }
```

Comment:
are we going to add finance storage too https://github.com/folio-org/mod-orders-storage/blob/80a80ecf09b8f90fe6c6d64e9193d099a6e81fd8/descriptors/ModuleDescriptor-template.json#L666

Reply:
Here is releted ticket for this: https://folio-org.atlassian.net/browse/MODORDSTOR-213
In TenantReferenceAPI I see that we don't have calls to finance-storage, also I don't see this SELECT statement that was described in that ticket:
SELECT jsonb -> 'code'
FROM ${myuniversity}_mod_finance_storage.fund

I think we can just delete that module permission

Reply:
Here we had ticket to remove finance-storage.funds dependency:
https://folio-org.atlassian.net/browse/MODORDSTOR-226
And module permission was not removed at that time, so removing now, thanks @azizbekxm for this catch